controllers/userController.js

```diff
@@ -0,0 +1,78 @@
+const models = require('../models')
+const bcryptjs = require('bcryptjs')
+const jwt = require('jsonwebtoken')
+const user = require('../models/user')
+
+const register = async (req, res) => {
+    try {
+        const checkEmail = await models.User.findOne({ where: { email: req.body.email } })
+        if (checkEmail) throw new Error ("User already exists")
+        const hashedPassword = await bcryptjs.hash(req.body.password, 10)
+
+        const payload = {
+            name: req.body.name,
+            email: req.body.email,
+            password: hashedPassword
+        }
+        const newUser = await models.User.create(payload)
+        res.json({
+            isSuccess: true,
+            message: `Successfully registered ${newUser.name}`,
+            data: {
+                email: newUser.email
+            }
+        })
+        } catch (error) {
+        res.status(400).json({
+            isSuccess: false,
+            message: error.message,
+            data: null
+        })
+
+
+    }
+}
+
+const login = async (req, res) => {
+    try {
+        const userMatch = await models.User.findOne({ where: { email: req.body.email } })
+
+        if (!userMatch) throw new Error("User does not exist.")
+
+        const isValidPassword = await bcryptjs.compare(req.body.password, userMatch.password)
+
+        if (!isValidPassword) throw new Error("Invalid password")
+
+        const token = jwt.sign({
+            email: userMatch.email,
+            userId: userMatch.id,
+        }, process.env.JWT_KEY)
```

Comment:
You can import `JWT_KEY` from a config file instead where we can load the environment file and then export the value for use.

node is actually not loading your environment file here 🌚🌚🌚